### ISSUE TITLE
"context.getScope is not a function"

### ISSUE THREAD

**Issue by caseywatts** [43081j/eslint-plugin-wc#139]
When I ran eslint with this plugin's `flat/best-practice` settings enabled, I got this error:

```
Oops! Something went wrong! :(

ESLint: 9.9.1

TypeError: context.getScope is not a function
```

It was triggered by this rule:
```
Rule: "wc/no-child-traversal-in-connectedcallback"
```

This is probably related to some changes in eslint `9.0.0`

eslint.org has a page about this error specifically:
https://eslint.org/docs/latest/use/troubleshooting/v9-rule-api-changes

**Comment by 43081j** [comment]
This should be fixed by #137 but I'm not sure we released it yet 

I'll publish a new one this week if not

**Comment by nkabrown** [comment]
Just bumping this issue and flagging that ESLint v8.x end of life is this week on October 5th. Thanks.

https://eslint.org/blog/2024/09/eslint-v8-eol-version-support/

**Comment by 43081j** [comment]
Gah! So sorry this was forgotten. I'll release a new version tomorrow 👍

**Comment by honia19** [comment]
the same issue

**Comment by 43081j** [comment]
2.2.0 has been published 👍
